test/bills.test.js

```diff
@@ -0,0 +1,69 @@
+const chai                          = require('chai');
+const chaiHttp                      = require('chai-http');
+const app                           = require('../server');
+const createUserAdminAndLogin       = require('./utils/createUserAdminAndLogin');
+const {createBills, deleteAllBills} = require('./utils/createBills');
+
+chai.use(chaiHttp);
+chai.should();
+
+const expect = chai.expect;
+
+let credentials;
+
+describe('Bills', () => {
+    beforeEach(async () => {
+        await deleteAllBills();
+        credentials = await createUserAdminAndLogin();
+    });
+
+    describe('POST /api/v2/bills', () => {
+        it('Create bill and get it with the code', async () => {
+            const bill = await createBills();
+            const res  = await chai.request(app)
+                .post('/api/v2/bills')
+                .set('authorization', credentials.token)
+                .send({PostBody: {filter: {code: bill.code}, limit: 99}});
+            expect(res).to.have.status(200);
+            expect(res.body.datas[0].name).be.equals(bill.name);
+        });
+        it('Create bill and get it with the id - w/o authentication', async () => {
+            const bill = await createBills();
+            const res  = await chai.request(app)
+                .post('/api/v2/bills')
+                .send({PostBody: {filter: {_id: bill._id}, limit: 99}});
+            expect(res).to.have.status(403);
+            expect(res.body).have.property('code');
+            expect(res.body.code).to.be.equal('AccessUnauthorized');
+        });
+        it('Create bill and get it with the id - w/o the good id', async () => {
+            await createBills();
+            const res = await chai.request(app)
+                .post('/api/v2/bills')
+                .set('authorization', credentials.token)
+                .send({PostBody: {filter: {_id: '111111111111111111111111'}, limit: 99}});
+            expect(res).to.have.status(200);
```

Comment:
Weird, I expect to have a 404 not a 200 when sending wrong information.
If the api send weird thing don't follow blindly what the api returns to you

Reply:
Yes. Big change for that, don't change it please